bouter/free/__init__.py

```diff
@@ -14,6 +14,19 @@ def n_tail_segments(self):
     def n_fish(self):
         return self["tracking+fish_tracking"]["n_fish_max"]
 
+    @property
+    def get_scale_mm(self):
```

Comment:
it is a property, so the get_ prefix is unnecessary. you can make it more descriptive and call it camera_px_in_mm